architecture/core_concepts/deployments.adoc

```diff
@@ -14,33 +14,32 @@ toc::[]
 
 == Replication Controllers
 
-A replication controller ensures that a specified number of replicas of a pod
-are running at all times. If pods exit or are deleted, the replica controller
-acts to instantiate more up to the desired number. Likewise, if there are more
-running than desired, it deletes as many as necessary to match the number.
+A
+https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/replication-controller.md[replication
+controller] ensures that a specified number of replicas of a pod are running at
+all times. If pods exit or are deleted, the replication controller acts to
+instantiate more up to the defined number. Likewise, if there are more running
+than desired, it deletes as many as necessary to match the defined amount.
 
-The definition of a replication controller consists mainly of:
+A replication controller configuration consists of:
 
 1. The number of replicas desired (which can be adjusted at runtime).
-2. A pod definition for creating a replicated pod.
+2. A pod definition to use when creating a replicated pod.
 3. A selector for identifying managed pods.
 
-The selector is just a set of labels that all of the pods managed by the
-replication controller should have. So that set of labels is included
-in the pod definition that the replication controller instantiates.
-This selector is used by the replication controller to determine how many
+A selector is a set of labels assigned to
+the pods that are managed by the replication controller. These labels are
+included in the pod definition that the replication controller instantiates.
+The replication controller uses the selector to determine how many
 instances of the pod are already running in order to adjust as needed.
 
-It is not the job of the replication controller to perform auto-scaling
-based on load or traffic, as it does not track either; rather, this
-would require its replica count to be adjusted by an external auto-scaler.
+The replication controller does not perform auto-scaling based on load or
+traffic, as it does not track either. Rather, this would require its replica
+count to be adjusted by an external auto-scaler.
 
-Replication controllers are a core Kubernetes object, `*ReplicationController*`.
-The
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/replication-controller.md[Kubernetes documentation]
-has more details on replication controllers.
+Replication controllers are a core Kubernetes object called `*ReplicationController*`.
```

Comment:
Gotcha. Changed to: 

"A replication controller is a core Kubernetes object..."